app/components/file-details/key-insights/index.hbs

```diff
@@ -0,0 +1,55 @@
+<div class='p-3'>
+  <AkTypography>
+    Based on the comparison with previous
```

Comment:
Non translated text.

Reply:
Please ignore this for now this will be handled when key insights is ready for release